File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] tmpArr = s.split(" ");
        int[] arr = new int[tmpArr.length];
        for(int i = 0; i < arr.length; i++){
            arr[i] = Integer.parseInt(tmpArr[i]);
        }
        Arrays.sort(arr);
        answer = "" + arr[0] + " " + arr[arr.length - 1];
        return answer;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    # 문자열을 공백으로 나눠 리스트로 변환
    tmp_arr = list(map(int, s.split(" ")))
    
    # 리스트 정렬
    tmp_arr.sort()
    
    # 최소값과 최대값을 문자열로 연결
    answer = f"{tmp_arr[0]} {tmp_arr[-1]}"
    return answer
